src/components/ResendVerification/ResendVerification.tsx

```diff
@@ -1,22 +1,55 @@
-import React from "react"
+import React, { useState } from "react"
 import "./ResendVerification.scss"
+import axios, { AxiosRequestConfig, AxiosResponse } from "axios"
+import Recaptcha from "react-google-recaptcha"
```

Comment:
I had to change this to `import ReCAPTCHA from "react-google-recaptcha"` when I first built it, but this seems to work fine now... not sure what was going on there...

Reply:
yeah me neither. I updated it to ReCAPTCHA anyway